readme.md

```diff
@@ -13,8 +13,9 @@
 
 Jotai is pronounced "joe-tie" and means "state" in Japanese.
 
-You can try a live demo [here](https://codesandbox.io/s/jotai-demo-47wvh)
-and [there](https://codesandbox.io/s/jotai-demo-forked-x2g5d).
+You can try live demos in the following: 
+* [Demo 1](https://codesandbox.io/s/jotai-demo-47wvh)
+* [Demo 2](https://codesandbox.io/s/jotai-demo-forked-x2g5d).
```

Comment:
Nice suggestion. Would you make it a single line somehow?

Reply:
Done 👍